index.html

```diff
@@ -654,7 +654,7 @@ <h3>Directory Service API</h3>
                     The HTTP API responses must use appropriate status codes described in 
                     this section for success and error responses.
                     <span class="rfc2119-assertion" id="tdd-http-errors"> 
-                        The HTTP API MUST use the Problem Details [[RFC7807]] format to carry
+                        The HTTP API SHOULD use the Problem Details [[RFC7807]] format to carry
```

Comment:
I'm not convinced this is a good idea.  Don't we want to be clear what the data schema of all data returned by the directory is?  What is the justification for allowing implementations to use alternative error formats?  Note: I think this should be a MUST, but I think it would be ok (regardless of what RFC7807 says, I need to check) to allow *extensions* of the problem details format, e.g. implementation-dependent extra fields.

Reply:
Replied below.